lib/src/intl_suggestors/intl_migrator.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:collection/collection.dart' show IterableExtension;
+import 'package:file/file.dart';
+import 'package:over_react_codemod/src/intl_suggestors/utils.dart';
+import 'package:over_react_codemod/src/util/component_usage.dart';
+import 'package:over_react_codemod/src/util/component_usage_migrator.dart';
+
+class IntlMigrator extends ComponentUsageMigrator {
+  final File _outputFile;
+  final String _className;
+
+  IntlMigrator(this._className, this._outputFile);
+
+  @override
+  String get fixmePrefix => 'FIXME(intl_migration)';
+  @override
+  bool get shouldFlagUnsafeMethodCalls => false;
+  @override
+  bool get shouldFlagUntypedSingleProp => false;
+  @override
+  bool get shouldFlagRefProp => false;
+  @override
+  bool get shouldFlagClassName => false;
+  @override
+  bool get shouldFlagExtensionMembers => false;
+  @override
+  bool get shouldFlagPrefixedProps => false;
+
+  @override
+  bool shouldMigrateUsage(FluentComponentUsage usage) =>
+      usage.cascadedProps.any((prop) =>
+          isValidStringLiteralProp(prop) ||
+          isValidStringInterpolationProp(prop)) ||
+      usage.children.any((child) =>
+          isValidStringLiteralNode(child.node) ||
+          isValidStringInterpolationNode(child.node));
+
+  @override
+  void migrateUsage(FluentComponentUsage usage) {
+    super.migrateUsage(usage);
+    final namePrefix =
+        (usage.node.thisOrAncestorOfType<ClassDeclaration>()?.name ??
+                usage.node.thisOrAncestorOfType<VariableDeclaration>()?.name)
+            .toString();
```

Comment:
I think it'd be clearer here if the null case wasn't handled using `toString`
```suggestion
            ?.name ?? 'null';
```